### DISCUSSION TITLE
After update dnscrypt-proxy to 2.1.1...

### DISCUSSION THREAD

**Discussion by Zovereign** [DNSCrypt/dnscrypt-proxy#2080] [Q&A]
After update dnscrypt-proxy to 2.1.1...
Why am I seeing this....
![Screenshot 2022-04-03 005717](https://user-images.githubusercontent.com/31466405/161395870-37e80726-a160-4b7a-84dd-6d8dc79302a9.png)
...which I don't normally see it?

Thank you in advanced.

**Comment by bcookatpcsd** [comment]
Not sure where that output is coming from.. 

But gravity is something to do with pihole? 

Do you query dnscrypt-proxy directly?

What does dnscrypt-proxy give you as an error.. it could be blocking the ipv6/AAAA lookup.. 

block_ipv6 = false
(default is false.. not to block)

**Reply by Zovereign** [reply]
Thank you for your reply, I am not sure either. I never visit the site as well. So, I have no clue at all. Anyways, it's not a big deal after all. 

Thank you very much 😉
